packages/beacon-node/src/chain/archiveStore/utils/archiveBlocks.ts

```diff
@@ -212,8 +218,15 @@ async function migrateBlobSidecarsFromHotToColdDb(
     const canonicalBlobSidecarsEntries: KeyValue<Slot, Uint8Array>[] = await Promise.all(
       canonicalBlocks
         .filter((block) => {
-          const blkSeq = config.getForkSeq(block.slot);
-          return blkSeq >= ForkSeq.deneb && blkSeq < ForkSeq.fulu;
+          const blockSlot = block.slot;
+          const blockEpoch = computeEpochAtSlot(blockSlot);
+          const forkSeq = config.getForkSeq(blockSlot);
+          return (
+            forkSeq >= ForkSeq.deneb &&
+            forkSeq < ForkSeq.fulu &&
+            // if block is out of ${config.MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS}, skip this step
+            blockEpoch >= currentEpoch - config.MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS
```

Comment:
this is main fix, refer to this in [unstable](https://github.com/ChainSafe/lodestar/blob/6dd10cbe48f61cf53e1cf01404d8b96386736df5/packages/beacon-node/src/chain/archiveStore/utils/archiveBlocks.ts#L185)

Reply:
is this change missing because of pending rebase?

Reply:
@matthewkeil good catch, just added that

Reply:
@g11tech I can't trace which specific commit was it but think it's just merge/rebase issue